source/docs/fedora-atomic-workstation.md

```diff
@@ -18,14 +18,14 @@ dd if=yourisoname.iso of=/dev/sdb bs=4M
 
 **Installing inside an existing system**
 
-If you rather want to install FAW inside an existing system, please follow the steps described on [https://pagure.io/workstation-ostree-config/blob/master/f/README-install-inside.md](Pagure).
+If you rather want to install FAW inside an existing system, please follow the steps described on [Pagure](https://pagure.io/workstation-ostree-config/blob/master/f/README-install-inside.md).
 
 ## Support
 
-For now, you can ask us questions in the #atomic channel on IRC Freenode or via Twitter [https://twitter.com/projectatomic](@projectatomic). You can also use the [atomic-devel](https://lists.projectatomic.io/mailman/listinfo/atomic-devel) mailing list. Contributions are welcome.
+For now, you can ask us questions in the #atomic channel on IRC Freenode or via Twitter [@projectatomic](https://twitter.com/projectatomic). You can also use the [https://lists.projectatomic.io/mailman/listinfo/atomic-devel](atomic-devel) mailing list. Contributions are welcome.
```

Comment:
so, now we have undone #520, can you fix it back to `[atomic-devel](https://lists.projectatomic.io/mailman/listinfo/atomic-devel)`

Reply:
😃